frontend/src/components/RatingBars.tsx

```diff
@@ -1,13 +1,22 @@
 import { FC, useState } from 'react';
 import classNames from 'classnames';
 import { Rating as MaterialRating } from '@mui/material';
+import { Trans } from 'react-i18next';
 import { TaskRatingDimension } from '../../../shared/types/customTypes';
 import { BusinessIcon, WorkRoundIcon } from './RoleIcons';
 import css from './RatingBars.module.scss';
 import { revertScale, convertScale } from '../../../shared/utils/conversion';
 
 const classes = classNames.bind(css);
 
+const labels: { [index: string]: string[] } = {
+  1: ['Not Relevant', 'Trivial'],
+  2: ['A Bit Important', 'Simple'],
+  3: ['Somewhat Important', 'Moderate'],
+  4: ['Important', 'Complex'],
+  5: ['Business Critical', 'Extremely Complex'],
+};
```

Comment:
```suggestion
const labels: { [K in TaskRatingDimension]: { [index: string]: string } } = {
  [TaskRatingDimension.BusinessValue]: {
    1: 'Not Relevant',
    2: 'A Bit Important',
    3: 'Somewhat Important',
    4: 'Important',
    5: 'Business Critical',
  },
  [TaskRatingDimension.Complexity]: {
    1: 'Trivial',
    2: 'Simple',
    3: 'Moderate',
    4: 'Complex',
    5: 'Extremely Complex',
  },
};
```
This might be a bit more understandable if the rating dimensions are mentioned explicitly.
(and swapping the list's dimensions makes that easier)

Reply:
Right that is 100% better. Fixed